Source/Dafny/Resolver.cs

```diff
@@ -13550,12 +13550,13 @@ private class RBranchExpr : RBranch {
           var it = outFormal.Type;
           Type st = SubstType(it, typeMap);
           var lhs = s.Lhs[i];
+          var outIndex = callee.Outs.Count == 1 ? "" : $" {i}";
```

Comment:
Maybe change `{i]` into `at index {i}` ?

Reply:
I was thinking about it too. It does not make sense to say "argument 0" because usually we don't start counting at zero indeed !
So precising that it's an index is nice so that it explains why the number is zero-based.

Instead of lengthening with "at index 0", why don't we just increase the positional argument by 1, so that it is easier to decipher? Are you ok with this alternative?

Reply:
@RustanLeino will be very sad if you start counting at 1 ^^

Reply:
And @RustanLeino and I have continued to have the friendliest and slowest argument about this in the entire time I have known him. :D